final_task/rss_reader/scripts/parser_rss.py

```diff
@@ -0,0 +1,134 @@
+import json
+import re
+import signal
+import logging
+from contextlib import contextmanager, closing
+from urllib.error import URLError
+import feedparser
+from dateutil import parser
+from final_task.rss_reader.scripts import News
+import html
+
+
+class TimeOutExeption(Exception): pass
+
+
+module_logger = logging.getLogger("rss_reader.scripts.parser_rss")
+
+
+@contextmanager
+def timeout_sec(seconds):
+    """
+       contextmanager to check the expectation of a response
+       and if the response does not come for a long time, an error
+    """
+
+    def signal_handler(signum, frame):
+        raise TimeOutExeption(Exception('Time out'))
+
+    signal.signal(signal.SIGALRM, signal_handler)
+    signal.alarm(seconds)
+    try:
+        yield
+    finally:
+        signal.alarm(0)
```

Comment:
Try to check some standard methods of `feedparser` to check connection or timeout instead of this context manager